python/lsst/cell_coadds/_fits.py

```diff
@@ -136,6 +137,7 @@ def readAsMultipleCellCoadd(self) -> MultipleCellCoadd:
     def _readSingleCellCoadd(
         data: Mapping[str, Any],
         common: CommonComponents,
+        header: Mapping[str, Any],  # TODO: Use a more specific type here.
```

Comment:
Shouldn't "TODOs" have a ticket associated with them, with more details?

Reply:
And there's another one above,L90: `units=CoaddUnits(1), # TODO: read from FITS TUNIT1`